doc/source/cluster/index.rst

```diff
@@ -16,7 +16,7 @@ done so often), but the real power is using Ray on a cluster of machines.
 
 Ray can automatically interact with the cloud provider to request or release
 instances. You can specify :ref:`a configuration <cluster-config>` to launch
-clusters on :ref:`AWS, GCP, Azure, Kubernetes, Aliyun, on-premise, or even on
+clusters on :ref:`AWS, GCP, Azure (community managed), Kubernetes, on-premise, or even on
```

Comment:
Somewhat important nit: I think we should mention that either everythin (except k8s) is community supported, or not mention it, as it's implied by the fact that it's implied by the maintainer list on this page: https://docs.ray.io/en/latest/cluster/cloud.html